tendrl/node_agent/message/handler.py

```diff
@@ -57,7 +58,13 @@ def bind_unix_listener(self):
             self.sock.setblocking(0)
             self.sock.bind(socket_path)
             self.sock.listen(50)
-        except (TypeError, BlockingIOError, socket_error, ValueError) as ex:
+            gid = pwd.getpwnam("tendrl-user").pw_gid
```

Comment:
@nthomas-redhat This user is required for collectd to run a custom plugin on new notification because of its(collectd's) limitation that it can't run a plugin on notification(threshold breach detected) as root.
Also, please check the my comment above...